egi_notebooks_hub/egispawner.py

```diff
@@ -152,62 +150,82 @@ class DataHubSpawner(EGISpawner):
                 host""",
     )
 
-    manager_class = Unicode(
-        "eginotebooks.manager.MixedContentsManager",
+    force_proxy_io = Bool(False, config=True, help="""Force the use of proxied I/O""")
+
+    force_direct_io = Bool(False, config=True, help="""Force the use of direct I/O""")
```

Comment:
i think it's better to set it to True as default ( i didn't see you added it to the conf)

Reply:
this will make that spaces that are not supported will fail, which is not nice

Reply:
mm, so you think that the oneclient is going to use direct i/o or proxy automatically depending on the space?

Reply:
that would be the ideal, is there a way to check that?

Reply:
i think we should avoid the proxy i/o , so users wil access only  spaces which are supported locally by the oneprovider colocated with the notebooks

Reply:
ok, should we try to fix this in this PR, or maybe focus on that for another one?

Reply:
i think we can keep the configuration like this, then do tests and see what is best default:-)